docs/experience/webvitals.md

```diff
@@ -0,0 +1,55 @@
+title: Web Vitals
+description: Gathering information about web vitals - essential site health metrics
+
+Supported by Chrome, the [Web Vitals](https://web.dev/vitals/) is a Google initiative to provide unified guidance for metrics that are crucial for delivering the best user experience for websites and webapps. 
+
+<img
+  class="content-modal-image"
+  alt="Web Vitals Overview"
+  src="../../images/experience/webvitals/webvitals.png"
+  title="Web Vitals Overview"
+/>
+
+### Core Web Vitals
+
+Both [Sematext Experience](/experience/) and [Sematext Synthetics](/synthetics/browser-monitor/#web-vitals) support the Core Web Vitals out of the box and include the following metrics:
+
+ * **Largest Contentful Paint (LCP)** - measures *page loading* performance. Google suggests that **LCP** should be **below 2500 milliseconds**, which means that to provide good user experience, the largest contentful paint should occur within 2.5 seconds of when the page first starts loading.
```

Comment:
If we write these definitions as sections with anchors then we can easily link to them, say from PU, or from blog posts, or from SC itself.

Reply:
Good idea, done.